README.md

```diff
@@ -100,7 +100,9 @@ ALLOW_FUTURE_SUBMISSION_DATE
 ```
 
 Set ALLOW_FUTURE_SUBMISSION_DATE to true to allow integration tests to run with submission dates that are in the future.
-A copy of the `.env` file including the current values can be found in the `Shared-LAA` section of LastPass.
+A copy of the `.env` file including the current values can be found in the `Shared-LAA` section of LastPass (reach out to the team if you don't have access).
+
+Note: Remove “GOOGLE_SHEETS_” from several of the variables in .env file
```

Comment:
Did you download the correct file?  I cannot see any `GOOGLE_SHEETS_` lines in the CFE .env file.
There _are_ some in the *Apply* file though 😕